src/infrastructure/content/ServerPort.java

```diff
@@ -1,11 +1,27 @@
-package infrastructure.content;
-
-public class ServerPort implements IServerPort {
-	/*
-	 * This method will save the port of the board server locally to send message over networking
-	 */
-	@Override
-	public void sendPort(int port) {
-		
-	}
+package infrastructure.content;
+
+/**
+ * 

```

Comment:
Give a small description. One line for simple descriptions is fine.

Reply:
Ok, i will do it.